src/components/FeatureSelector/Defaults.tsx

```diff
@@ -53,7 +58,8 @@ const options: FeatureSliderData[] = [
     max: 100,
     currentMin: 0,
     currentMax: 100,
-    units: '%'
+    units: '%',
+    description : 'Speechiness detects the presence of spoken words in a track'
```

Comment:
```suggestion
    description : 'Speechiness detects the presence of spoken (not sung) words in a track'
```

I don't think this is clear enough. The documentation says that the first third is normal songs, last third is podcasts and middle third is a mix (e.g. rap)